mindocr/data/rec_lmdb_dataset.py

```diff
@@ -15,20 +14,16 @@ class LMDBDataset(BaseDataset):
     The annotaiton format is required to aligned to paddle, which can be done using the `converter.py` script.
 
     Args:
-        is_train: whether the dataset is for training
-        data_dir: data root directory for lmdb dataset(s)
-        shuffle: Optional, if not given, shuffle = is_train
-        transform_pipeline: list of dict, key - transform class name, value - a dict of param config.
-                    e.g., [{'DecodeImage': {'img_mode': 'BGR', 'channel_first': False}}]
-            -       if None, default transform pipeline for text detection will be taken.
-        output_columns (list): optional, indicates the keys in data dict that are expected to output for dataloader.
-            if None, all data keys will be used for return.
-        filter_max_len (bool): Filter the records where the label is longer than the `max_text_len`.
-        max_text_len (int): The maximum text length the dataloader expected.
+        is_train: Whether the dataset is for training. Default: True.
```

Comment:
@zhtmike we can remove `is_train` and set `shuffle` to False by default. There's no reason to shuffle training set on initialization if the loader shuffles it on every iteration. We can keep the `shuffle` parameter for cases when a training set is partially used (`sample_ratio` < 1).

Reply:
Agree, we can remove it in the next PR